lib/tasks/data/supplier_locations.yml

```diff
@@ -229,7 +399,7 @@ suppliers:
     - CAM4
     - CAM5
     - CDI
-    - CITWMC

```

Comment:
`CITWMC` is Westminster Mag Court and should not have been in this list as its a court